en/docs/administer/key-managers/configure-global-key-manager.md

```diff
@@ -0,0 +1,124 @@
+# Configure the Global Key Manager
+
+This document outlines the steps to enable the global key manager support in WSO2 API Manager (API-M). With this feature, a single token generated using the global key manager can be used to invoke cross-tenant APIs without generating a token in that specific tenant's key manager.
```

Comment:
Let's ensure consistency in the capitalization of 'global key manager' throughout the document.  Please use either 'global key manager' or 'Global Key Manager' consistently.

Reply:
fixed